tables/afr-za-g2.ctb

```diff
@@ -0,0 +1,850 @@
+# liblouis: Afrikaans Braille Code (Grade 2)
+#
+# -----------
+#-name: Afrikaans braille, grade 2
+#-display-name: Afrikaans (UBC) contracted braille
+#
+#+locale:af
+#+type:literary
+#+contraction:full
+#+grade:2
+#+system:ueb
+#
+# TODO: Please correct the metadata above. It is not meant to be
+# accurate nor complete. It hasn't been verified by the table
+# author yet. It is merely an attempt by the liblouis maintainers
+# to get some sensible initial values in place.
+#
+# TODO: Please add a reference to official documentation about
+# the implemented braille code. Preferably submit the documents
+# to https://protect-eu.mimecast.com/s/1hhmCLvY2h0n9JtB1qp0?domain=github.com.
+# -----------
+#
+#  Copyright (C) 2019 South African Braille Authority 
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <https://protect-eu.mimecast.com/s/FsveCMJE2hp43QIka-Cf?domain=gnu.org>.
```

Comment:
Again problem with the link. Should be `<http://www.gnu.org/licenses/>`.

Reply:
I believe I fixed this url